tests/test_core_configure.py

```diff
@@ -129,6 +132,174 @@ def test_verify_spec_succeeds(capsys):
     core_configure.verify_spec(spec)
 
 
+def test_verify_spec_fails_multiple_roles_own_schema(capsys):
+    """
+    Should successfuly detect config that defines two roles as an owner of a single schema.
+    """
+    spec_yaml = """
+    jfinance:
```

Comment:
[nitpick] I'd indent all of your triple-quoted strings so it's easier to see where the variable starts and ends.

In addition, it might be worth removing some of the parts of the specs that aren't actually being tested here (e.g. `is_superuser` here, other things elsewhere).